app/database.ts

```diff
@@ -383,9 +377,9 @@ export class Database_View
     }
 
     async addEvent_v2(event: Event_V2, url: URL) {
-        console.log(event, url);
         this.events_v2.set(event.id, event);
-        await this.recordRelay(event.id, url.toString());
+        // some space urls hava a pathname, example: wss://example.com/nostr/space
+        await this.recordRelay(event.id, url.origin + url.pathname);
```

Comment:
Not a blocker:
`addEvent` v1 also uses `recordRelay`.

Change recordRelay to take a `URL` type